lib/public/views/lhcPeriods/ActiveColumns/lhcPeriodsActiveColumns.js

```diff
@@ -33,6 +33,7 @@ export const lhcPeriodsActiveColumns = {
                 { lhcPeriodName: name },
                 { class: `btn mh1 ${runsCount > 0 ? 'bg-light-blue' : ''}` },
             ),
+            frontLink('data passes', 'data-passes-per-lhc-period-overview', { lhcPeriodId: id }, { class: 'btn mh1' }),
```

Comment:
Data passes with captial first letter no?

Reply:
So then, also, need to make runs button with capital letter

Reply:
Probably